src/components/MonitorListFilter.vue

```diff
@@ -162,23 +162,26 @@ export default {
         Status,
         Tag,
     },
-    props: {
-        filterState: {
-            type: Object,
-            required: true,
-        }
-    },
     emits: [ "updateFilter" ],
     data() {
         return {
             tagsList: [],
+            filterNames: [
+                "status",
+                "active",
+                "tags",
+            ],
         };
     },
     computed: {
         numFiltersActive() {
             let num = 0;
 
-            Object.values(this.filterState).forEach(item => {
+            Object.values(
```

Comment:
I am unclear on why you did you add this filtering step

Reply:
Because the filter state was kept in `filterState`. Now it is kept in the URL, so we need to parse the URL query and filter the queries that are used for filters (status, active, tags)

Reply:
Okay, lets simplify this to

```js
return this.$router.currentRoute.value.query.status?.length || 0 +
    this.$router.currentRoute.value.query.active?.length || 0 +
    this.$router.currentRoute.value.query.tags?.length || 0;
    

```